main.cpp

```diff
@@ -113,6 +124,76 @@ Thing 4) chord
     3) be omitted
  */
 
+/*
+Thing 5) Keys
+5 properties:
+    1) is key depressed
+    2) velocity of key pressed (1-127)
+    3) MIDI number of note pressed (1-127)
+    4) Is note legato (note held, while another is played)
+    5) Is keyboard set to monophonic
+3 things it can do:
+    1) play legato
+    2) play chords
+    3) play monophonic
+ */
+
+/*
+Thing 6) Oscillator
+5 properties:
+    1) pitch value of oscillator
+    2) octave setting of oscillator (int 'foot' selection of LFO, 32, 16, 8, 4, 2)
+    3) is oscillator on
+    4) is oscillator controlled by pitch via keys
```

Comment:
instead of `is ____`
try:
`oscillator volume level`
and `oscillator pitch source`
the first one can be represented with a `float` or `double`
and the 2nd can be represented with a `std::string`
- [ ] revise